server/server.go

```diff
@@ -48,6 +49,7 @@ func (s *Server) listenAndServe() {
 	}
 
 	s.port = listener.Addr().(*net.TCPAddr).Port
+	signal.SendMediaServerStarted(s.port)
```

Comment:
This is triggered inside the `Server.listenAndServe` function, this will trigger for all types of `Server` including the `PairingServer`

Reply:
Hi @Samyoul , thanks for pointing out, just fixed.